release_notes/370-release-notes.adoc

```diff
@@ -168,6 +169,17 @@ In the table, features are marked with the following statuses:
 [id="bug-fixes-3670"]
 == Bug fixes
 
+[id="fixed-in-version-3702"]
+=== Resolved in version 3.70.2
+
+Release date: 5 October 2022:
```

Comment:
There is an extra colon here, would you want to fix it before I merge, or after?